parsers/TW.py

```diff
@@ -96,6 +97,7 @@ def fetch_production(
         "production": {
             "coal": coal_production,
             "gas": gas_production,
+            "geothermal": production.loc["Geothermal"].capacity,
```

Comment:
Was this meant to be `production.loc["Geothermal"].output` ?

Reply:
It may have been but right now that would have been undefined as there is no Geothermal "class" in the dataset.